ai-integrations/langgraph.ipynb

```diff
@@ -100,7 +104,8 @@
    "source": [
     "# Use helper method to create the vector search index\n",
     "vector_store.create_vector_search_index(\n",
-    "   dimensions = 1536 # The dimensions of the vector embeddings to be indexed \n",
+    "   dimensions = 2048, # The dimensions of the vector embeddings to be indexed \n",
+    "   wait_until_complete = 60 # Number of seconds to wait for the index to build (can take around a minute)\n",
```

Comment:
I can't comment on the line where I'm getting an error, but this file is failing in ln 190 below where we're defining a full-text search tool. I'm getting the error `NameError: name 'collection' is not defined`. It looks like we're defining a collection in ln 126 below _within_ the scope of the `create_fulltext_search_index` method call. I think we need to move the `collection` initialization to a new variable under the client, in ln 123, to make it available in ln 190 below.